src/action.ts

```diff
@@ -26,6 +26,7 @@ export default async function main() {
   const dryRun = core.getInput('dry_run');
   const customReleaseRules = core.getInput('custom_release_rules');
   const shouldFetchAllTags = core.getInput('fetch_all_tags');
+  const commit_sha = core.getInput('commit_sha');
```

Comment:
could you use camelCase to match the style of the rest of the code?

Reply:
Oh yes, sure, sorry!